pythonforandroid/toolchain.py

```diff
@@ -847,7 +847,9 @@ def apk(self, args):
 
                 # gradle output apks somewhere else
                 # and don't have version in file
-                apk_dir = join(dist.dist_dir, "build", "outputs", "apk")
+                apk_dir = join(dist.dist_dir,
+                               "build", "outputs", "apk",
+                               args.build_mode)
```

Comment:
Just find out that this line breaks buildozer APK searchs. :[

Reply:
Buildozer was searching in the apk directory, not a subdirectory.

Reply:
Ei @tito, it seems that @AndreMiras already fixed that in buildozer's master branch, see:

kivy/buildozer#766